dev-python/meson-python/meson-python-0.17.1.ebuild

```diff
@@ -32,6 +32,7 @@ RDEPEND="
 "
 BDEPEND="
 	>=dev-python/cython-0.29.34[${PYTHON_USEDEP}]
+	>=dev-python/pyproject-metadata-0.7.1[${PYTHON_USEDEP}]
```

Comment:
Given it calls `meson` and `ninja` too, I think it should just take the whole `${RDEPEND}` here.

Reply:
It seems that `meson` and `ninja` are present in the base stage3 images but i tested by renaming each binary, and emerge does indeed fail, so the fact that emerging `meson-python` succeeds is an accident that something else has pulled those into the base stage3 images.